packages/astro/src/runtime/server/index.ts

```diff
@@ -603,10 +603,7 @@ export async function renderHead(result: SSRResult): Promise<string> {
 			if ('data-astro-component-hydration' in script.props) {
 				needsHydrationStyles = true;
 			}
-			return renderElement('script', {
-				...script,
-				props: { ...script.props, 'astro-script': result._metadata.pathname + '/script-' + i },
-			});
+			return renderElement('script', script);
```

Comment:
Note that this is not related to this change, just noticed that we were still adding these special attributes. They were used by the legacy build system but are not needed any more.